src/app/dts/template-editor/template-editor.component.ts

```diff
@@ -58,7 +58,8 @@ export class TemplateEditorComponent implements OnInit {
       templateData: new FormControl(),
       templateKey: new FormControl(),
       templateId: new FormControl(),
-      author: new FormControl()
+      author: new FormControl(),
```

Comment:
Note: `formBuilder.group` accepts a shorter syntax for this:
```
this.formBuilder.group({
      templateData: '',	  
      templateKey: '',
      templateId: '',	     
      author: ''	
});
```

Reply:
Applied the recommended change.